### PR TITLE
Load comments from Datastore and return in JSON format

### PR DESCRIPTION
**Pull request by fatimazali**
Screenshot 2020-07-09 at 12 33 24 AM
- Load comments, sorted by descending timestamps (display them while starting from the most recent comment) 
- Comments remain after restarting the webpage and re-running the dev server

### CODE REVIEW

**Review by jessimb — APPROVED** [review]
LGTM - thanks for keeping PRs small!

**Comment by fatimazali** [comment]
Added 4 additional commits to enable user to delete comments from the DOM and Datastore

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -16,20 +16,60 @@
...
+/** Servlet that parses user comments, stores them in Datastore, and retrieves them from Datastore. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  /**
+  * Fetches limited number of comments from Datastore and returns them
+  * @param request the request message
+  * @param response the response message
+  * @throws IOException if an I/O error occurs
+  */
```
[the remark this answers is not in the document]
Comment (fatimazali) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:43]: noted, thank you!

[the remark this answers is not in the document]
Comment (fatimazali) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:43]: (formatting resolved in PR 11)

**Pull request merged**